pkg/volumereplica/volumereplica.go

```diff
@@ -126,36 +128,36 @@ func PoolNameFromCVR(cvr *cstor.CStorVolumeReplica) string {
 
 // ToDo: Move this to backup package
 
-//// PoolNameFromBackup gets the name of cstorpool from cstorvolumereplica label
-//// if not found then gets cstorpoolinstance name from the OPENEBS_IO_POOL_NAME
-//// env
-//func PoolNameFromBackup(bkp *cstor.CStorBackup) string {
-//	poolname := bkp.Labels[CStorPoolUIDKey]
-//	if strings.TrimSpace(poolname) == "" {
-//		poolname = os.Getenv(string("OPENEBS_IO_POOL_NAME"))
-//		if strings.TrimSpace(poolname) == "" {
-//			return ""
-//		}
-//	}
-//	return PoolPrefix + poolname
-//
-//}
+// PoolNameFromBackup gets the name of cstorpool from cstorvolumereplica label
+// if not found then gets cstorpoolinstance name from the OPENEBS_IO_POOL_NAME
+// env
+func PoolNameFromBackup(bkp *openebsio.CStorBackup) string {
+	poolname := bkp.Labels[CStorPoolUIDKey]
+	if strings.TrimSpace(poolname) == "" {
+		poolname = os.Getenv(string("OPENEBS_IO_POOL_NAME"))
+		if strings.TrimSpace(poolname) == "" {
+			return ""
+		}
+	}
+	return PoolPrefix + poolname
+
+}
 
 // ToDo: Move this to restore package
 
-//// PoolNameFromRestore gets the name of cstorPool from cstorvolumereplica label
-//// if not found then gets cstorPoolInstance name from the OPENEBS_IO_POOL_NAME
-//// env
-//func PoolNameFromRestore(rst *apis.CStorRestore) string {
-//	poolname := rst.Labels[CStorPoolUIDKey]
-//	if strings.TrimSpace(poolname) == "" {
-//		poolname = os.Getenv(string("OPENEBS_IO_POOL_NAME"))
-//		if strings.TrimSpace(poolname) == "" {
-//			return ""
-//		}
-//	}
-//	return PoolPrefix + poolname
-//}
+// PoolNameFromRestore gets the name of cstorPool from cstorvolumereplica label
+// if not found then gets cstorPoolInstance name from the OPENEBS_IO_POOL_NAME
+// env
+func PoolNameFromRestore(rst *openebsio.CStorRestore) string {
+	poolname := rst.Labels[CStorPoolUIDKey]
+	if strings.TrimSpace(poolname) == "" {
+		poolname = os.Getenv(string("OPENEBS_IO_POOL_NAME"))
+		if strings.TrimSpace(poolname) == "" {
+			return ""
+		}
+	}
+	return PoolPrefix + poolname
+}
```

Comment:
Change it to below snippet because in latest CSPI controller we are fetching pool name from ENV `OPENEBS_IO_POOL_NAME`
```go
func getPoolName() string {
		poolname = os.Getenv(string("OPENEBS_IO_POOL_NAME"))
		if strings.TrimSpace(poolname) == "" {
			return ""
		}
	return PoolPrefix + poolname
}
```